Bottom: 00fcba5835039daa960abd8b5fcd2abbc53d9cb9
Top:    83f1bde3bd461116ddce5071eb66282bda7fbe98
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 03:03:31 +0100

parse_y.y: Try to avoid two shift/reduce and reduce/reduce warnings

TODO: Check this matches up against our expected behavior.

Removing the parsedata section ("I think") has no effect, as elements
hit the alternative | element clause within parsepcb.

HOWEVER... I need to check whether there is any other valid syntax
inside an element file, or any backwards compatibility with old formats
which we may miss with the loss of parsedata.


---

diff --git a/src/parse_y.y b/src/parse_y.y
index 46058e3..29db9df 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -706,7 +706,6 @@ pcbdefinition
 		| rats
 		| layer
 		| element
-		| error { YYABORT; }
 		;
 
 via
